recipes/polymorphic_value/all/conanfile.py

```diff
@@ -32,38 +34,44 @@ def _minimum_compilers_version(self):
         }
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        # FIXME: workaround https://github.com/conan-io/conan/issues/12210
+        info = self if Version(conan_version).major < 2 else self.info
```

Comment:
No workaround, please. It should be fixed on next releases.

Reply:
reverted